testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks/delta.properties

```diff
@@ -1,8 +1,10 @@
 connector.name=delta_lake
 hive.metastore=glue
 hive.metastore.glue.region=${ENV:AWS_REGION}
-fs.hadoop.enabled=true
+fs.hadoop.enabled=false
+fs.native-s3.enabled=true
 # We need to give access to bucket owner (the AWS account integrated with Databricks), otherwise files won't be readable from Databricks
-hive.s3.upload-acl-type=BUCKET_OWNER_FULL_CONTROL
+s3.canned-acl=BUCKET_OWNER_FULL_CONTROL
+s3.region=us-east-1
```

Comment:
Why do we need to set the region here but not for the others?

Let's move this above the comment, so that it's clear that the comment is only for the ACL and not the region

Reply:
That's a testing left-over.

Reply:
I _think_ the tests failed without the region.. I cant remember if it was in the CI or local. I m trying to test this after removing region now. lets see

Reply:
tests passed. removed the region